aiida_vasp/utils/fixtures/data.py

```diff
@@ -227,13 +232,21 @@ def ref_retrieved_nscf():
     return retrieved
 
 
-@pytest.fixture
+@pytest.fixture()
 def vasprun_parser():
     """Return an instance of VasprunParser for a reference vasprun.xml."""
     file_name = 'vasprun.xml'
     path = data_path('vasprun', file_name)
-    parser = VasprunParser(path)
+    parser = VasprunParser(file_path=path)
+    return parser
 
+
+@pytest.fixture(params=['basic'])
+def vasp_xml(request):
```

Comment:
This name is ambiguous, it could also be a fixture for an xml file. I would suggest: `vasprun_parser_variants` or similar.

Also please adapt the docstring to not be a copy of `vasprun_parser()`'s

Reply:
Merged with the existing.